src/components/KDropdownMenu/KDropdownItem.vue

```diff
@@ -158,11 +158,11 @@ const availableComponents = computed((): DropdownItemRenderedRecord => ({
 li.k-dropdown-item {
   align-items: center;
   display: flex;
-  font-size: 16px;
+  font-size: var(--type-md);
```

Comment:
Can we provide fallback values when using CSS variables?

Reply:
😢  CSS variables must _always_ have a fallback value. If you want to utilize the Sass equivalent, you can do this:
```scss
font-size: var(--type-md, type(md));
```